app/views/pages/privacy.html.slim

```diff
@@ -0,0 +1,87 @@
+h1 Privacy Policy
+
+em Last updated: May 25th, 2018
+
+h2 Introduction
+p We take the protection of our users (“User/you/your”) personal data very seriously and strictly comply with applicable data protection laws and regulations. In our privacy policy below (“Privacy Policy”) we provide you with an overview of what data we collect for what purpose and how we ensure the protection of the data. 
```

Comment:
Just outta curiosity: are these _other_ `"` signs on purpose?

Reply:
ah — this is a habit I have when writing “plain text“, I always use the keyboard shortcut for the “correct” ones when not in a code context. Forgot this would end up here eventually 😅